core-bundle/assets/controllers/message-outlet-controller.js

```diff
@@ -0,0 +1,19 @@
+import { Controller } from '@hotwired/stimulus';
+
+export default class extends Controller {
+    static targets = ['message'];
+
+    messageTargetConnected(el) {
+        if(!el.querySelector('button.close')) {
+            setTimeout(() => this._hide(el), 5000);
```

Comment:
We'll have to discuss the A11Y aspect of this. I think as long as the messages remain in the DOM and are thus accessible for screen readers, it is okay to hide them after 5 seconds. But I would appreciate it if an expert could confirm this. Otherwise we would have to add an option to disable hiding flash messages automatically.

Reply:
> or the user has ticked the new checkbox "do not hide messages" in their profile

I missed this, sorry 🙈